pkg/vulnsrc/k8svulndb/k8svulndb.go

```diff
@@ -0,0 +1,162 @@
+package k8svulndb
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+	"time"
+
+	bolt "go.etcd.io/bbolt"
+	"golang.org/x/xerrors"
+	"io"
+	"log"
+	"path/filepath"
+
+	"github.com/aquasecurity/trivy-db/pkg/db"
+	"github.com/aquasecurity/trivy-db/pkg/types"
+	"github.com/aquasecurity/trivy-db/pkg/utils"
+	"github.com/aquasecurity/trivy-db/pkg/vulnsrc/bucket"
+	"github.com/aquasecurity/trivy-db/pkg/vulnsrc/vulnerability"
+)
+
+const (
+	vulnListDir = "vuln-list-k8s"
+	apiDir      = "k8s/cves"
+)
+
+var (
+	source = types.DataSource{
+		ID:   vulnerability.K8sVulnDB,
+		Name: "The k8s Vulnerability Database",
```

Comment:
nit: They call it "Official CVE Feed" or "Official Kubernetes CVE Feed".
https://kubernetes.io/docs/reference/issues-security/official-cve-feed/
https://kubernetes.io/blog/2022/09/12/k8s-cve-feed-alpha/

```suggestion
		Name: "Official Kubernetes CVE Feed",
```